ml_frameworks_bot/utils.py

```diff
@@ -51,5 +54,28 @@ def make_embedding_model(embedding_model_name: str) -> BaseEmbedding:
             "text-embedding-3-large",
             "text-embedding-ada-002",
         ]
-        else HuggingFaceEmbedding(model_name=embedding_model_name)
+        else HuggingFaceEmbedding(
+            model_name=embedding_model_name,
+            trust_remote_code=True,
+            model_kwargs={"torch_dtype": torch.float16},
```

Comment:
`model_kwargs` should be a optional parameter for the function. 

Example usage:
```python
embedding_model = make_embedding_model("text-embedding-3-small", **hf_kwargs)
```